src/dmd/objc.d

```diff
@@ -131,6 +133,20 @@ Objc objc()
     return _objc;
 }
 
+
+/**
+ * This struct contains all data for a class declaration that is needed for the
```

Comment:
remove `This struct contains`. It's redundant.

Reply:
Ok.